.github/workflows/ibis-tpch-queries.yml

```diff
@@ -0,0 +1,42 @@
+# vim: filetype=yaml
+name: TPC-H
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.repository }}-${{ github.head_ref || github.sha }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  tpch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: install python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+
+      - run: python -m pip install --upgrade pip click sqlparse
+
+      - name: install ibis
+        run: python -m pip install ".[sqlite]"
```

Comment:
leaving a reminder here to add `duckdb` to the extras list after the 3.0.0 release.